x/dclauth/types/constants.go

```diff
@@ -16,9 +16,9 @@ package types
 
 // Default parameter values.
 const (
-	MaxMemoCharacters             uint64  = 256
-	TxSizeCostPerByte             uint64  = 10
-	DefaultSigVerifyCostED25519   uint64  = 590
-	DefaultSigVerifyCostSecp256k1 uint64  = 1000
-	AccountApprovalPercent        float64 = 0.66
+	DclMaxMemoCharacters      uint64  = 256
```

Comment:
shouldn't we refer to [default cosmos'](https://github.com/cosmos/cosmos-sdk/blob/2f335a2b8cc4fcfa337c104823fab9f6ecb387da/x/auth/types/params.go#L13) value here (as for `DefaultTxSigLimit`)